demo/full/scripts/controllers/ContentList.tsx

```diff
@@ -144,18 +150,8 @@ function constructContentList(): Partial<Record<string, IDisplayedContent[]>> {
   );
 }
 
-/**
- * Generate URL with hash-string which can be used to reload the page with the
- * current stored content or demo content. This can be used for example to
- * share some content with other people.
- * Returns null if it could not generate an URL for the current content.
- * @param {Object} content - The content object as constructed in the
- * ContentList.
- * @param {Object} state - The current ContentList state.
- * @returns {string|null}
- */
-function generateLinkForContent(
-  content: IDisplayedContent,
+function generateConfigForContent(
```

Comment:
The previous JSDoc could have been lightly edited to better understand its purpose